packages/@atjson/source-html/src/converter/third-party-embeds.ts

```diff
@@ -214,6 +215,22 @@ export default function convertThirdPartyEmbeds(doc: Document) {
         })
       );
     });
+  /**
+   * CNE Ticketing Widget
+   */
```

Comment:
Can we please update this comment to describe what this HTML structure is? IE, this is a _custom_ HTML element used to describe an event registration block with a specific URL, with some specific metadata encoded in the URL

Reply:
I'll add a short description, thanks

Reply:
Updated comment with a short description